packages/ui/src/models/visualization/flows/support/camel-component-default.service.ts

```diff
@@ -123,6 +123,51 @@ export class CamelComponentDefaultService {
           steps: []
         `);
 
+      case 'setHeader':
+        return parse(`
+        setHeader:
+          id: ${getCamelRandomId('setHeader')}
+          expression:
+            simple:
+              expression: ""
+        `);
+
+      case 'setProperty':
+        return parse(`
+        setProperty:
+          id: ${getCamelRandomId('setProperty')}
+          expression:
+            simple:
+              expression: ""
+        `);
+
+      case 'setVariable':
+        return parse(`
+        setVariable:
+          id: ${getCamelRandomId('setVariable')}
+          expression:
+            simple:
+              expression: ""
+        `);
+
+      case 'setBody':
+        return parse(`
+        setBody:
+          id: ${getCamelRandomId('setBody')}
+          expression:
+            simple:
+              expression: ""
+        `);
+
+      case 'filter':
+        return parse(`
+        filter:
+          id: ${getCamelRandomId('filter')}
+          expression:
+            simple:
+              expression: ""
+          steps: []
+        `);
```

Comment:
```suggestion
      case 'setHeader':
      case 'setProperty':
      case 'setVariable':
      case 'setBody':      
      case 'filter':
        return parse(`
        [processorName]:
          id: ${getCamelRandomId(processorName)}
          expression:
            simple:
              expression: {}
        `);
```

Reply:
What do you think about the suggestion above?

Having an `expression: {}` will also cause the source code to be highlighted to have a pending configuration

Reply:
Update: Having `expression: {}` forces the expression TextField to show `[object Object]`, my bad.

PR to fix it: https://github.com/KaotoIO/kaoto/pull/1077
Related issue: https://github.com/KaotoIO/kaoto/issues/1076